roles/grafana_agent/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# defaults file for agent
+
+# Mimir URL and creds
+agent_mimir_url: "http://sepia-grafana.front.sepia.ceph.com:9009/api/v1/push"
+agent_mimir_username: "admin"
+
+# Enable metrics globally
+agent_metrics: true
```

Comment:
Why does this conditional exist? Do you expect that certain hosts will set this to `false` in a future PR?

Reply:
I understand. My point is that in your PR, it's always `true`. Why not simply remove it?